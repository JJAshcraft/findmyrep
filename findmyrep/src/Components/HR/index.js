import styled from "styled-components";

const HR = styled.hr`
  border: 1px solid #eee;
`;

export default HR;
