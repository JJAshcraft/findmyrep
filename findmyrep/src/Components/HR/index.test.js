import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import HR from "./index.js";

test("HR component renders with proper styling", () => {
  const tree = renderer.create(<HR />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("border", "1px solid #eee");
});
