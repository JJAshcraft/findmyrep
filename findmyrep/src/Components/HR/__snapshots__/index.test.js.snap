// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HR component renders with proper styling 1`] = `
.c0 {
  border: 1px solid #eee;
}

<hr
  className="c0"
/>
`;
